src/main/java/com/sap/oss/phosphor/fosstars/data/github/LicenseInfo.java

```diff
@@ -356,7 +356,8 @@ public static void main(String... args) throws Exception {
             + "disallowedLicensePatterns:\n"
             + "  - API\n"
             + "repositoryExceptions:\n"
-            + "  - https://github.com/SAP/SapMachine\n",
+            + "  - https://github.com/SAP/SapMachine\n"
```

Comment:
Why don't we maintain this list at one place as some constants or so? So that it can be reused at places like below.  In the future, we just need to modify this list at one place.